crates/solidity/inputs/language/bindings/rules.msgb

```diff
@@ -1060,3 +1065,253 @@ attribute symbol_reference = symbol  => type = "push_symbol", symbol = symbol, i
 @funcall [Expression [FunctionCallExpression @args [ArgumentsDeclaration]]] {
   edge @args.lexical_scope -> @funcall.lexical_scope
 }
+
+
+;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;
+;;; Yul
+;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;
+
+;;; Blocks and statements
+
+@block [YulBlock] {
+  node @block.lexical_scope
+  ; Variables defined in this block (only used to forward the init block
+  ; declarations in a for statement)
+  node @block.defs
+  ; Function definitions accessible from the block (ie. defined in the block, or
+  ; accessible in the enclosing parent block)
+  node @block.fundefs
```

Comment:
nit suggestion: WDYT of `@block.function_defs` instead? trying to keep track of all the abbreviations in the ever-increasing file.